storage/storage.cpp

```diff
@@ -770,6 +771,11 @@ void Storage::RegisterCountryFiles(TIndex const & index, string const & director
   TLocalFilePtr localFile = GetLocalFile(index, version);
   if (localFile)
     return;
+  // @TODO(bykoianko) This place should be implemented faster.
+  // You use Find twice. Here and in GetCountryFile.
+  SimpleTree<Country> const * node = m_countries.Find(Country(index));
```

Comment:
Yes! It cannot happened until data (mwm and countries.txt) are correct. 
Thanks! The errors in data are found. This code are removed.